libs/base/System.idr

```diff
@@ -62,7 +64,7 @@ time = do MkRaw t <- foreign FFI_C "idris_time" (IO (Raw Integer))
           pure t
 
 ||| Specify interval to sleep for, must be in range [0, 1000000]
```

Comment:
We are restricting values above 1000000 because that results in unexpected behaviour on some platforms. The API works as expected in range [0, 1000000]. Let me know if you want to change it to (0, 1000000] or ]0, 1000000] (haven't read this notation before).